CHANGES.rst

```diff
@@ -23,6 +23,10 @@ New Features
   - Added ``PrecessedGeocentric`` frame, which is based on GCRS, but precessed
     to a specific requested mean equinox. [#3758]
 
+  - `SphericalRepresentation` now has a `._unitrep` class attribute to specify 
+    an equivalent UnitSphericalRepresentation. This allows subclasses of 
+    representations to pair up correctly.
```

Comment:
I would move this to the "API changes" section then, if nothing else, since it's more of an internal API feature.

Also some nitpicks--use double-backticks insgead of singe-backticks, and add a reference to the PR # :)